ios/Swift/README.md

```diff
@@ -1,17 +1,66 @@
-# Image detection with iOS device photos
+# Cloud Vision Swift Sample
 
 This app demonstrates how to use the [Cloud Vision API](https://cloud.google.com/vision/) to run label and face detection on an image.
 
-## Local Setup
+## Prerequisites
+- An API key for the Cloud Vision API (See
+  [the docs][getting-started] to learn more)
+- An iOS device running iOS 9.0 or above
+- [Xcode 7][xcode]
```

Comment:
Is it still the case that Windows is not a supported platform at all for ios dev?  If so, we should probably put that in the prereqs as a formality, e.g. 'an osx machine or emulator'